davinci/src/main/kotlin/com/pingidentity/davinci/CollectorRegistry.kt

```diff
@@ -47,5 +50,9 @@ internal class CollectorRegistry : ModuleInitializer() {
 
         CollectorFactory.register("SINGLE_SELECT", ::SingleSelectCollector)
         CollectorFactory.register("MULTI_SELECT", ::MultiSelectCollector)
+
+        CollectorFactory.register("DEVICE_REGISTRATION", ::DeviceRegistrationCollector)
```

Comment:
I would suggest to create constants for the collectors

Reply:
The string is not shared, may not create a constant for this.
I will review constant handling for Android SDK later.